database/migrations/2019_09_28_171753_config_fix.php

```diff
@@ -175,7 +176,7 @@ public function up()
 				'key' => 'sorting_Albums_col',
 				'value' => 'max_takestamp',
 				'cat' => 'Gallery',
-				'type_range' => 'id|title|description|public|max_takestamp|min_takestamp',
+				'type_range' => 'id|title|description|public|max_takestamp|min_takestamp|created_at',
```

Comment:
I'm still not convinced that this is a good idea. It's still a big edge case.

Reply:
Well, we do not propose it visually. ;)